portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -36,16 +37,23 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    //Load the entity by kind 
+
+    int max = 3;
+    int userChoice = getUserChoice(request);
```

Comment:
userChoice is not very descriptive. commentCount, maxLimit, or maxCount would be better.

Reply:
Thank you! I will change it.

Reply:
> userChoice is not very descriptive. commentCount, maxLimit, or maxCount would be better.

Thank you! I will change it.